### DISCUSSION TITLE
Proposal: Incentivized AKT Lending Pool on Nitron

### DISCUSSION THREAD

**Discussion by samyukta1512** [akash-network/community#322] [Governance Proposals]
### Goal

Akash Network is one of the top networks in Cosmos, topping the charts in IBC volume on a weekly basis.

[Nitron](https://app.dem.exchange/nitron), ([Demex’s](https://dem.exchange/) in-app money market) had the privilege of being the first money market on Cosmos to support a number of Cosmos native tokens such as ATOM, EVMOS, OSMO, LSDs, LP tokens, and so on as collateral. 

Demex is looking to build a partnership with Akash Network that is aimed at benefitting the Akash community by whitelisting AKT (Akash Network’s native token) as collateral on Nitron. We envision a long-term partnership with Akash Network by exploring and collaborating on more DeFi areas in the future.

### Summary

This proposal is aimed to bootstrap the lending of AKT on Demex to further increase utility by allowing AKT to be whitelisted as collateral to borrow other assets for more capital efficiency. It is asking for a one-time transfer of 6,000 AKT tokens (~$5,000) from the community pool across, as incentives only to Demex’s lending pool for AKT collateral, in order to raise awareness and increase demand for AKT.

This forum post is to discuss incentivizing the AKT lending pool on Nitron with AKT tokens, to further increase the utility, buying pressure, and adoption of the AKT token.

### What is Demex and Nitron

[Demex](https://app.dem.exchange/) (or Decentralized Mercantile Exchange) is the cross-chain derivatives DEX powered by [Carbon](https://carbon.network/) and is designed to support any type of financial asset imaginable. Demex aims to make finance accessible and trust-optional while providing an intuitive and familiar user experience.

[Nitron](https://app.dem.exchange/nitron) is the name of Demex's lending and borrowing money market. It works similarly to AAVE and supports a wide range of networks and assets. It also has a public liquidation platform, which allows anyone to perform permissionless liquidations on undercollateralized loans via the UI.

Demex exists on Carbon Network, a DeFi-focused L1 Cosmos Network built with Cosmos SDK and is IBC-enabled. The team behind Carbon Network is called Switcheo Labs, a doxxed Singapore Web3 development studio founded in 2018.

### Benefits

1. Since launching, Akash has seen a strong and steady increase in on-chain activity market cap and capabilities, giving its native token AKT a lot of traction. We wish to incentivize the AKT pool to directly provide more yield opportunities for the Akash community.
2. This incentive will encourage users to lend out their AKT tokens instead of selling it if a user is in need of cash or liquidity. This will maintain the market cap of AKT and reduce the selling pressure while continuing to generate fees for Akash Network.
3.  Depositing AKT tokens onto Nitron allows it to be used as borrowing power and unlock additional capital. This means users can borrow other tokens such as stablecoins against AKT tokens, and use the borrowed tokens to perform other DeFi activities. 
4. Users who view AKT as undervalued and are bullish can also deposit AKT to borrow USD to buy SOMM and repeat to achieve a leverage-long position.

**Additional details:**

As both chains are IBC-enabled, AKT deposits to Nitron will be done via Demex’s deposit UI which uses IBC transfers which are a lot more secure.

Demex will engage in marketing efforts throughout the incentivized period, to constantly promote the ongoing campaign, and to inform more AKT holders of this opportunity to earn yield.

### AKT Collateral Money Market Parameters

- Oracle: The price oracle will reference the floor value of the AKT price on Osmosis.
- Loan-to-value (LTV): 50%
- Liquidation threshold: 55%
- Liquidation bonus: 15%
- Supply cap: 130,000 QTY (about $110,000 - can be raised anytime)
- Borrow cap: 0 (will not be borrowable when incentives are live to prevent looping)
- Optimal utilization: 50%

### Projections

1. If the incentive attracts a TVL of around $100,000 of AKT, it would give an incentive APR of about 20% APR over 90 days which is competitive in the current yield farming landscape while still being higher than AKT staking rewards to make it worthwhile.
2. If a user wants to earn even more yield, a user can go leverage long on AKT by depositing $1k of AKT to borrow $500 of USD at around 3% APR to buy more AKT on Osmosis and deposit to Nitron. Doing this loop once would generate an estimated yield of above 50% APY on the initial $1k AKT deposit.

### Distribution

1. This program will distribute the authorized amount of tokens over a period of 90 days, starting from the launch of the AKT lending pool and collateral.
2. 100% of the AKT incentives will be distributed to the Switcheo team to be deployed pro rata to users on Demex who deposit AKT tokens in Nitron’s lending contract via Carbon’s external incentive feature on its Collateralized Debt Position (CDP) module.
3. This proposal, if accepted, will spend $5,000 worth of AKT tokens from the community pool, to the dev team address at **akash1t8lz3zjv9cyd8x4h060w3hhdnzagm5cugztd4n** on Akash Network in order to fund the external incentive module on Nitron according to the parameters of this proposal.

In the future, Akash may want to launch similar programs with the [Switcheo team](https://www.switcheo.com/) with different parameters - any future program is outside of the scope of this governance action and will require a new proposal.

Consolidated reference links:
https://docs.google.com/document/d/1St77B6gykRFiuxOfAKouhvZyOfd10LyjZmi0CnJlFNA/edit

**Comment by hewi333** [comment]
Hi samyukta1512,

Thank you for your interest in Akash and the time it took to put this together. I have a few points I'd like to address.

1. You say, "Since launching, Akash has seen a strong and steady increase in on-chain activity and TVL" but there is no TVL in Akash as we are not a DeFi network. 
2. You say you had the privilege of being the first money market on Cosmos and that AKT is not listed on any lending protocols. I do see AKT listed on at least one other lending protocol, however it is not incentivized.

I'm not sure how this proposal advances the development or use of decentralized compute. AKT is permissionless and anyone can incorporate it into their product/protocols but to me this is just paying incentives to attract TVL to Demex. Can you go more into detail on how this helps Akash at large vs just users speculating on token price?

**Reply by samyukta1512** [reply]
Hi Heyhewi! Thank you for taking the time to engage in an active discussion for our proposal, I’m more than happy to address your questions. 

1. Apologies for the first point, this was a typo and has been amended in the proposal above. The point we intend to make is that Akash Network has seen a strong increase in on-chain activity, market cap, and capabilities.
2. Acknowledging the presence of the other lending markets, this has been removed from the proposal as well. 

This will enable the Akash community, to earn yield and borrow other assets when in need of liquidity without having to sell off their tokens. This helps maintain liquidity, reducing pressure, and bringing about another token utility. Furthermore, being on a platform like Nitron will bring more exposure to it from other Cosmos users.

**Reply by Aihonaken** [reply]
i'm not much familiar with Demex and Nitron, but on estimate how many cosmos users are using Demex and Nitron?

**Comment by Aihonaken** [comment]
This is a nice proposal, though AKT can be lent on Kava as well. I like the prop, i just don't see the benefit after the one time incentive after the 90 days. I see the benefits for Nitron, i don't see much for Akash though.

**Reply by samyukta1512** [reply]
Hi Aihonaken! Thank you for your support. 

Apologies and we acknowledge the presence of other lending markets, this has been removed from the proposal. 

We intend to propose this as a means of beginning a long-term strategic partnership with the Akash Network. By listing AKT on Nitron, it will receive a lot of exposure from the users on our platform, from a variety of other IBC ecosystems. 

AKT lending goes beyond just the lending incentives. This gives AKT holders the means to leverage their portfolio to borrow other assets and stablecoins. We believe this is a very strong value proposition, as we constantly have lending and borrowing incentive programs running on Nitron. We intend to do more with Akash Network in the future such as listing spot/perpetual markets, supporting liquidity pools, host trading competitions and so on. We would like to begin this partnership with an incentivized lending pool.

**Reply by hewi333** [reply]
@samyukta1512 Can you go into some metrics on Nitron? number of users, TVL, etc.

Knowing the number of active users is important for understanding the type of exposure this gets AKT

**Comment by piber-dev** [comment]
this proposal is interesting only if it is accurate that the staking rewards will be lower than the LP rewards. as if the staking rewards are lower than the LP rewards, there will be no incentive for holders of $AKT to add it to the LP rather than simply staking them. i also like the ability to lend out $AKT for other coins depending on the details of this lending system.

**Comment by SophoclesSummer** [comment]
This proposal was rejected on 11/03/2023 by a 38% majority. 
https://www.mintscan.io/akash/proposals/235
